opal/mca/reachable/reachable.h

```diff
@@ -75,6 +75,8 @@ typedef int (*opal_reachable_base_module_fini_fn_t)(void);
  * interfaces, but only a (comparable) reachability between any pair
  * of local/remote interfaces.
  *
+ * This function will show an identical local and remote interface as
+ * reachable through loopback.
```

Comment:
Nit pick: it will show it as reachable -- I don't think the "reachable" framework indicates / implies which interface the traffic will actually go through, right?

Reply:
That's true, I'll change it to just say "as reachable".